applications/StructuralMechanicsApplication/tests/response_function_tests/rectangular_plate_parameters_nonlinear.json

```diff
@@ -0,0 +1,111 @@
+{
+    "problem_data"             : {
+        "problem_name"    : "rectangular_plate_structure",
+        "start_time"      : 0.0,
+        "end_time"        : 1.0,
+        "parallel_type"   : "OpenMP",
+        "echo_level"      : 0
+    },
+    "solver_settings"          : {
+        "solver_type"                        : "static",
+        "echo_level"                         : 0,
+        "time_stepping"                      : {
+            "time_step" : 0.1
+        },
+        "analysis_type"                      : "non_linear",
+        "model_part_name" : "rectangular_plate_structure",
+        "domain_size"     : 3,
+        "model_import_settings"              : {
+            "input_type"       : "mdpa",
+            "input_filename"   : "rectangular_plate_structure",
+            "input_file_label" : 0
+        },
+        "material_import_settings"           : {
+            "materials_filename" : "StructuralMaterials.json"
+        },
+        "line_search"                        : false,
+        "convergence_criterion"              : "residual_criterion",
+        "displacement_relative_tolerance"    : 0.0001,
+        "displacement_absolute_tolerance"    : 1e-9,
+        "residual_relative_tolerance"        : 0.0001,
+        "residual_absolute_tolerance"        : 1e-9,
+        "max_iteration"                      : 10,
+        "problem_domain_sub_model_part_list" : ["Parts_AREAS"],
+        "processes_sub_model_part_list"      : ["DISPLACEMENT_DIR_BC","PointLoad3D_NODES"],
+        "linear_solver_settings"       : {
+            "solver_type"         : "Super_LU"
```

Comment:
same

Reply:
Sorry for that, these files are not needed at all, I will remove them.